www/class/centreon-clapi/centreon.Config.Poller.class.php

```diff
@@ -596,7 +596,7 @@ public function sendTrapCfg($pollerId = null)
             throw new CentreonClapiException(self::MISSING_POLLER_ID);
         }
         $this->testPollerId($pollerId);
-        $centreonDir = CentreonUtils::getCentreonDir();
+        $centreonDir = realpath(__DIR__ . "/../../../");
```

Comment:
Why don't you use the class property $this->centreon_path ?
```suggestion
        $centreonDir = $this->centreon_path;
```